packages/graphql-server/src/resolvers/negotiators/types.graphql

```diff
@@ -0,0 +1,179 @@
+"""
+Request body used to create a new negotiator
+"""
+input CreateNegotiatorModelInput {
+  """
```

Comment:
remove redundant things
And you should check params required or not and put `!` to the schema

Reply:
This is a good point, rather than manually creating the schema, can we not use a generator lib to convert TS Definitions to GQL?